www/js/survey/multilabel/MultiLabelButtonGroup.tsx

```diff
@@ -114,7 +128,7 @@ const MultilabelButtonGroup = ({ trip, buttonsInline = false }) => {
             );
           })}
         </View>
-        {trip.verifiability === 'can-verify' && (
+        {verifiabilityForTrip(trip, timelineLabelMap[trip._id.$oid]) == 'can-verify' && (
```

Comment:
Theoretically, in a React project the difference should be negligible because React components are designed to only re-render when their state, or any stateful props, are explicitly updated. That's why React doesn't like mutations - if a value changes by any other way, the React component will not reflect the change because it doesn't know to re-render itself.
In contrast, I think the old Angular app was just constantly re-rendering things.

One caveat here though: the way `FlashList` works is complicated behind the scenes. It's a recycled list view where things DO have to get recomputed if they go off the screen and go back onto the screen. But there are things you can do to optimize this, which I've done my best of so far.
Some info: https://shopify.github.io/flash-list/docs/recycling https://shopify.github.io/flash-list/docs/fundamentals/performant-components/